docs/page-objects/items.rst

```diff
@@ -24,11 +25,46 @@ class. For example:
 Because itemadapter_ allows implementing support for arbitrary classes,
 any kind of Python object can potentially work as an item.
 
+Defining the item class of a page object class
+==============================================
+
+When inheriting from :class:`~.ItemPage`, indicate the item class to return
+between brackets:
+
+.. code-block:: python
+
+    @attrs.define
+    class MyPage(ItemPage[MyItem]):
+        ...
+
+To change the item class of a subclass that does not directly inherit from
+:class:`~.ItemPage`, use :class:`~.Returns`:
```

Comment:
If I'm not mistaken, technically you still can annotate any ItemPage subclass which hasn't been annotated with an item yet. For example, `WebPage[MyItem]` should work.